dev-db/etcd-wrapped/etcd-wrapped-0.4.9.ebuild

```diff
@@ -0,0 +1 @@
+files/etcd-wrapped.ebuild
```

Comment:
We can continue to build/install dev-db/etcd:0 until we finally have the guts to delete it.